src/Labs.Mvc/Helpers/Queries.cs

```diff
@@ -4,6 +4,7 @@ public static class Queries
         select * from openquery (sis, '
             select stvterm_code as code, stvterm_desc as description, stvterm_start_date as startdate, stvterm_end_date as enddate from stvterm
             where stvterm_end_date > sysdate
+            and (stvterm_trmt_code = ''Q' or stvterm_trmt_code = ''W'')
```

Comment:
@laholstege I think you are missing a single quote after Q, since they should be double single quotes to escape the openquery.